src/catacomb.rs

```diff
@@ -129,6 +133,12 @@ impl<B: Backend + 'static> Catacomb<B> {
         // XDG output protocol.
         xdg::init_xdg_output_manager(&mut display, None);
 
+        // Run user startup script.
+        if let Some(mut home) = dirs::home_dir() {
+            home.push(Path::new(ON_START_SCRIPT));
+            let _ = daemon::spawn_daemon("/bin/sh", iter::once(home.into_os_string()));
```

Comment:
```suggestion
            let _ = daemon::spawn_daemon("/bin/sh", [home.into_os_string()]);
```

Arrays should be the simplest way to get an `IntoIterator` going I think? Might need a reference, not sure.